### PR TITLE
add configuration example for Kea DHCPv4 server

### PR DESCRIPTION
**Pull request by mingshuoqiu**
Issue: https://github.com/harvester/harvester/issues/4618

### CODE REVIEW

File: docs/install/pxe-boot-install.md [hunk 1]
```diff
@@ -259,6 +259,83 @@ group {
 
 The config file declares a subnet and two groups. The first group is for hosts to boot using `CREATE` mode and the other one is for `JOIN` mode. By default, the iPXE path is chosen, but if it sees a PXE client it offers the iPXE image according to the client architecture. Please prepare those images and a TFTP server first.
 
+Since ISC announced the End of Life for the older ISC DHCP system in 2022, Kea DHCP is full-featured, open source, which is far newer, and is designed for a more modern network environment. For users who already use the Kea DHCPv4 server, please refer to the following example for the same configuration.
+For more information, see [Kea DHCPv4 Configuration](https://kea.readthedocs.io/en/kea-1.6.2/arm/dhcp4-srv.html#dhcpv4-server-configuration)
```
[the remark this answers is not in the document]
Comment (mingshuoqiu) [reply docs/install/pxe-boot-install.md:263]: Thanks. It looks more official.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

File: docs/install/pxe-boot-install.md [hunk 2]
```diff
@@ -259,6 +259,82 @@ group {
 
 The config file declares a subnet and two groups. The first group is for hosts to boot using `CREATE` mode and the other one is for `JOIN` mode. By default, the iPXE path is chosen, but if it sees a PXE client it offers the iPXE image according to the client architecture. Please prepare those images and a TFTP server first.
 
+The Internet Systems Consortium (ISC) announced the final [end-of-life (EOL) for ISC DHCP](https://www.isc.org/blogs/isc-dhcp-eol/) in 2022. ISC DHCP users are encouraged to migrate to the newer, feature-rich [Kea DHCP](https://www.isc.org/kea/), which the ISC designed for more modern network environments. If you are already using the Kea DHCPv4 server, check the following configuration example. For more information, see [Kea DHCPv4 Configuration](https://kea.readthedocs.io/en/kea-1.6.2/arm/dhcp4-srv.html#dhcpv4-server-configuration).
+```sh
```
Comment (starbops) [inline docs/install/pxe-boot-install.md:263]: Could we change this to `json`?

File: docs/install/pxe-boot-install.md [hunk 3]
```diff
@@ -319,6 +395,23 @@ group {
 
 The `elsif substring` statement is new, and it offers `http://10.100.0.10/harvester/ipxe.efi` when it sees a UEFI HTTP boot DHCP request. After the client fetches the iPXE program and runs it, the iPXE program will send a DHCP request again and load the iPXE script from the URL `http://10.100.0.10/harvester/ipxe-create-efi`.
 
+If you want to enable UEFI HTTP boot on the Kea DHCPv4 server, you must add a new `client-class` at the end of the `client-classes`.
+
+Example:
+```sh
```
Comment (starbops) [inline docs/install/pxe-boot-install.md:401]: Ditto

**Review by starbops — COMMENTED** [review]
Confirmed the setup works. LGTM. Just one nit. Thank you!

**Review by starbops — APPROVED** [review]

**Review by bk201 — APPROVED** [review]
Thanks for the update, we can consider add some example to https://github.com/harvester/ipxe-examples too.

**Pull request merged**